### ISSUE TITLE
3rd ThreadRipper support.

### ISSUE THREAD

**Issue by jasin755** [AMD-OSX/AMD_Vanilla#110]
3rd ThreadRipper support is missing.

**Comment by bigladadam** [comment]
From the AMD-OSX discord server: Attention all Users with TRX40 CPUs (Threadripper 3xxx). This chipset is currently not working with macOS. Everyone will be notified when it has been fixed.
I may be wrong but TRX40 support hasn't been added yet

**Comment by bpohvoodoo** [comment]
Any progress on this?

**Comment by jasin755** [comment]
> Any progress on this?

No. But  I found another solution https://www.macos86.it/topic/2509-guide-trx40-osx-bare-metal-proxmox-setup61-3

**Comment by nicklegger** [comment]
Wait so the past 2 days I have been wasting my time with my new 3970X? wtf
